frontend/src/App.scss

```diff
@@ -0,0 +1,31 @@
+.Home {
```

Comment:
css class (this is the same for scss) should be write in kebab-case or in camelCase, but not in PascalCase. Rename all of your class.